Formula/nef.rb

```diff
@@ -0,0 +1,36 @@
+class Nef < Formula
+  desc "💊 steroids for Xcode Playgrounds"
+  homepage "https://github.com/bow-swift/nef"
+  url "https://github.com/bow-swift/nef/archive/0.5.1.tar.gz"
+  sha256 "0c4331c15aa73056abc20eadd2e400ef1c949c02a7ebcebb1aaf78b84deb5cc8"
+
+  depends_on :xcode => "11.0"
+
+  def install
+    build_project
+
+    bin.install "./bin/nef"
+    bin.install "./bin/nef-common"
+    bin.install "./bin/nefc"
+    bin.install "./bin/nef-playground"
+    bin.install "./bin/nef-markdown-page"
+    bin.install "./bin/nef-markdown"
+    bin.install "./bin/nef-jekyll-page"
+    bin.install "./bin/nef-jekyll"
+    bin.install "./bin/nef-carbon-page"
+    bin.install "./bin/nef-carbon"
+    bin.install "./bin/nef-playground-book"
+  end
+
+  test do
+    print ""
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (`foo --version` or `foo --help`) are not sufficient, as explained in the [formula cookbook](https://docs.brew.sh/Formula-Cookbook).

In most cases, a good test would involve running a simple test case: run `#{bin}/foo input.txt`.
- Then you can check that the output is as expected (with `assert_equal` or `assert_match` on the output of `shell_output`)
- You can also check that an output file was created, if that is expected: `assert_predicate testpath/"output.txt", :exist?`

Some advice for specific cases:
- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
I have followed your comments and complete this test section. I have added tests for checking the business logic (units, integrations...) and others for the installed binaries - to check if the tool is working correctly.